openedx/features/course_experience/templates/course_experience/course-sock-fragment.html

```diff
@@ -16,48 +17,48 @@
         %endif
     >
         <button type="button" class="btn btn-primary focusable action-toggle-verification-sock">
-            Learn About Verified Certificates
+            ${_('Learn About Verified Certificates')}
         </button>
         <div class="verification-main-panel">
             <div class="verification-desc-panel content-main">
-                <h2>edX Verified Certificate</h2>
-                <h4>Why upgrade?</h4>
+                <h2>${_('{platform_name} Verified Certificate').format(platform_name=settings.PLATFORM_NAME)}</h2>
+                <h4>${_('Why upgrade?')}</h4>
                 <ul>
-                    <li>Official proof of completion</li>
-                    <li>Easily shareable certificate</li>
-                    <li>Proven motivator to complete the course</li>
-                    <li>Certificate purchases help edX continue to offer free courses</li>
+                    <li>${_('Official proof of completion')}</li>
+                    <li>${_('Easily shareable certificate')}</li>
+                    <li>${_('Proven motivator to complete the course')}</li>
+                    <li>${_('Certificate purchases help {platform_name} continue to offer free courses').format(platform_name=settings.PLATFORM_NAME)}</li>
                 </ul>
-                <h4>How it works</h4>
+                <h4>${_('How it works')}</h4>
                 <ul>
-                    <li>Pay the Verified Certificate upgrade fee</li>
-                    <li>Verify your identity with a webcam and government-issued ID</li>
-                    <li>Study hard and pass the course</li>
-                    <li>Share your certificate with friends, employers, and others</li>
+                    <li>${_('Pay the Verified Certificate upgrade fee')}</li>
+                    <li>${_('Verify your identity with a webcam and government-issued ID')}</li>
+                    <li>${_('Study hard and pass the course')}</li>
+                    <li>${_('Share your certificate with friends, employers, and others')}</li>
                 </ul>
-                <h4>edX Learner Stories</h4>
+                <h4>${_('edX Learner Stories')}</h4>
```

Comment:
since you're using platform_name above couldn't we just hide the whole learner stories section if platform name is not 'edX' or whatever our platform name is?

Reply:
Yeah, that's definitely a reasonable option. I'll add that. We'll still want these strings translated, though.